Peer testing eric (#453)
jpabadir/pace-c: documentation/peer-testing-2/peer-testing-report.md

**To be completed after the peer testing sessions**

#### High
- Server sometimes accidentally shut down for unexpected reason.

> - Need an extra tab for mentors to view the history of their declined mentee.

Reviewer: That's an interesting point. The client hasn't mentioned the ability to view declined mentees, so I don't  think it's an important aspect for the system, so maybe we could put that as a low priority, which could be a bonus feature for once we finalize the higher priority functionalities.

Author: Agree! Thank you for reviewing!

> - Mentor user should not to logout and login to update their suggested mentees

Reviewer: Minor typo: I think 'have' should be in between 'not' and 'to' 
```suggestion
- Mentor user should not have to logout and login to update their suggested mentees
```

### Testimonials from the Participants to Explain the Graph

- After finishing filling AdminForm and verfying new account, user was confused for what to do next because the page remained on 'verification email has been sent'. 
- User tried to fill the sample forms of mentor and mentee as an admin, which should not happen.

> - "what about google.com" user said when he noticed that he needed to enter an URL in avaliability field, which indicates that using tooltip to hint users is not enought and specific.

Reviewer: Minor typos
```suggestion
- "what about google.com" a user said when he noticed that he needed to enter a URL in the availability field, which indicates that using a tooltip to hint to users is not specific enough.
```
